docs-ref-conceptual/cheat-sheet-onboarding.md

```diff
@@ -0,0 +1,79 @@
+| How do I get a list of all commands, both core and extension? | For a list of command groups, in your console type `az` and <kbd>Enter</kbd>. For a list of subgroups and commands, use the `--help` parameter.  Example: `az account --help` or `az account subscription --help` will output a command list for just the group or subgroup. | [Reference index A-Z](/cli/azure/reference-index)
+| Does the Azure CLI have tab completion? | Yes!  When typing a command in PowerShell, press your <kbd>Tab</kbd> key. This feature is only available in PowerShell. | [Enable Tab Completion on PowerShell](/cli/azure/install-azure-cli-windows#enable-tab-completion-on-powershell)
+| How do I get in-line help? | Every command has a `--help` parameter that will result in descriptions and tips to display in your console. Example: `az account ---help` or `az vm create --help` | You can also [Request support](azure-cli-support-request.md).
+| How can I learn to use reference commands that cross multiple Azure services for a single job to be done? | We're glad you asked!  The Azure CLI provides several interactive commands that provide syntax and next steps.  In the next section, learn about `az interactive`, `az scenario`, `az next`, and `az find`. | [Learning Azure CLI](#learning-azure-cli)
+
+For side-by-side comparison examples of the Azure CLI and Azure PowerShell, see [Azure CLI vs Azure PowerShell: Side-by-side Command Comparison](/cli/azure/choose-the-right-azure-command-line-tool#azure-cli-vs-azure-powershell-side-by-side-command-comparison).
+
+See script comparisons between Bash, PowerShell and Cmd in [How to query Azure CLI command output using a JMESPath query](query-azure-cli.md).
+
+## Learning Azure CLI
+
+There are several reference commands that provide interactive prompts to help you learn the Azure CLI.
+
+| Command | Description | Example
+|-|-|-|
+| [az interactive](interactive-azure-cli.md) | In a PowerShell console, get auto-completion, command descriptions, and examples as you type | See [Work in interactive mode](interactive-azure-cli.md) for more information.  Example: Type `az interactive` in your console window then <kbd>Enter</kbd>.
+| [az scenario](/cli/azure/scenario) | Get end to end (E2E) examples and execute them with walk-through prompts. | Type `az scenario guide "keywords of interest"` in your console then <kbd>Enter</kbd>.  The Azure CLI will return scenarios based on your keyword(s).  Example: `az scenario guide "virtual machines"`
+| [az next](/cli/azure/reference-index#az-next) | Get recommendations for the next Azure CLI command based on customer execution patterns.  You will be prompted to select a command path. | Type `az next` in your console then <kbd>Enter</kbd>.  Pick from 1 of 7 preset command paths.
```

Comment:
"You will be prompted" should be changed to "You are prompted" for present tense